src/test/java/com/snowflake/kafka/connector/records/RecordContentTest.java

```diff
@@ -230,4 +231,44 @@ public void testConvertToJsonReadOnlyByteBuffer() {
     Schema schema = SchemaBuilder.bytes().build();
     assert RecordService.convertToJson(schema, buffer).toString().equals(expected);
   }
+
+  @Test
+  public void testSchematizationStringField() {
+    RecordService service = new RecordService();
+    SnowflakeJsonConverter jsonConverter = new SnowflakeJsonConverter();
```

Comment:
can you instead use JsonConverter in both the tests below? 

We are not using JsonConverter for `SNOWPIPE_STREAMING`